quickwit-search/src/service/rest.rs

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
why is this file called rest.rs!?

Reply:
I thought that `grpc_adapter.rs` in `quickwit-serve` was one of the `SearchServiceImpl`, so I moved it to `quickwit-search`. At that time, `service.rs` in `quickwit-search` that already existed was renamed to `rest.rs` (for warp) and `grpc_adapter.rs` (renamed to `grpc.rs`) (for tonic) was put in the same `service` module.

Reply:
I misunderstood. Let me take it back.

Reply:
Please see https://github.com/quickwit-inc/quickwit/pull/163/commits/47576c549c9b22cbb6e5f6aeb7c7d1ed1ba9ab9c